e2e/playwright/support/components/post.ts

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {ChannelsPost} from '@e2e-support/ui/components';
+
+// TODO: migrate this helpers to webapp's ChannelsPost after monorepo migration
+export const getBotTagFromPost = (post: ChannelsPost) => {
+    return post.container.locator('.post__header').locator('.BotTag', {hasText: 'BOT'});
+};
```

Comment:
I would just remove the todo comment. We will address this already when we extract the common code